<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidgetClass</class>
 <widget class="QWidget" name="LoginWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <widget class="QLabel" name="labelUsername">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;用户名&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;学生选课信息管理系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoginStu">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>学生登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegisterStu">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>学生注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoginAdmin">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>管理员登录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineUsername">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePassword">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelWarning">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LoginWidget.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnLoginAdmin</sender>
   <signal>clicked()</signal>
   <receiver>LoginWidgetClass</receiver>
   <slot>loginAdmin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLoginStu</sender>
   <signal>clicked()</signal>
   <receiver>LoginWidgetClass</receiver>
   <slot>loginStudent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRegisterStu</sender>
   <signal>clicked()</signal>
   <receiver>LoginWidgetClass</receiver>
   <slot>registerStudent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loginAdmin()</slot>
  <slot>loginStudent()</slot>
  <slot>registerStudent()</slot>
 </slots>
</ui>
